### ISSUE TITLE
NVDA doesn't keep focus mode in data grid

### ISSUE THREAD

**Issue by kolaps33** [nvaccess/nvda#15582]
### Steps to reproduce:
1. Open [aria data grid example](https://www.w3.org/WAI/ARIA/apg/patterns/grid/examples/data-grids/)
2. Run NVDA
3. Navigate to the "Example 1: Minimal Data Grid"
4. Navigate by arrow key into "description" column on the link 

### Actual behavior:
NVDA change to the browse mode and user cannot user arrow keys to continue navigation 

### Expected behavior:
NVDA should stay in focus mode in order to navigate by arrow keys to the next cells 

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
Version: 2023.3beta4 (2023.3.0.29485)

#### Windows version:
Version	10.0.22621 Build 22621

#### Name and version of other software in use when reproducing the issue:
Chrome Version 117.0.5938.132 (Official Build) (64-bit)

#### Does the issue still occur after restarting your computer?
yes 

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no 

#### If NVDA add-ons are disabled, is your problem still occurring?
yes

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
didn't try

**Comment by seanbudd** [comment]
Closing as duplicate of #13392
